File: app/src/main/java/com/example/facebookjava/Post.java
package com.example.facebookjava;

public class Post {
    private int profile;
    private String fullname;
    private int photo;
    public boolean isEditedView = false;
    boolean moreThan5 = false;


    public int getProfile() { return profile; }

    public String getFullname() {
        return fullname;
    }

    public int getPhoto() {
        return photo;
    }

    public Post(int profile, String fullname, int photo) {
        this.profile = profile;
        this.fullname = fullname;
        this.photo = photo;
    }

    public Post() {
        isEditedView = true;
    }

    public Post(boolean moreThan5) {
        this.moreThan5 = moreThan5;
    }


}

File: app/src/main/java/com/example/facebook/Post.kt
package com.example.facebook

class Post {
    var profile = 0
    var fullname: String? = null
    var photo = 0
    var isEditedView = false
    var moreThan5 = false

    constructor(profile: Int, fullname: String?, photo: Int) {
        this.profile = profile
        this.fullname = fullname
        this.photo = photo
    }

    constructor() {
        isEditedView = true
    }

    constructor(moreThan5: Boolean) {
        this.moreThan5 = moreThan5
    }
}
